### ISSUE TITLE
Can't add quests/notes not in the current map

### ISSUE THREAD

**Issue by Schaka** [AeroScripts/QuestieDev#38]
Currently, only notes for the current map are added through objectiveProcessors. 
This can lead to problems with quests that have objectives in different zones and is one of the reasons quests (until today) didn't show up in the tracker (or currentNotes) after logging into Ironforge with quests all over the world in your log.

We have to find a better way to do this. The current check is:
![image](https://cloud.githubusercontent.com/assets/2223171/6983173/11cabede-da1b-11e4-90aa-34112d506960.png)

If we just comment this out, we might flood the map with unnecessary notes. However, it can cause quite a few different bugs, if we just leave it as is.

**Comment by AeroScripts** [comment]
Gotta be careful though, rendering all points for all different maps could cause craziness for commonly dropped items (ex wool cloth.) Obviously some sort of limit would need to be in place, the hard part would be getting it to populate the map with that max number outwards from the player. (sorting functions on large numbers of objects like that could cause horrible lag)

**Comment by Schaka** [comment]
Maybe we could just store notes in the Cartographer SavedVariables and it would automatically draw them, if the user switches to that map. I think the important point is having the notes in our own SavedVariables too, because we need it for the tracker frame to properly show quests in other zones when you're starting to track in, say, Ironforge.

We also need it for proper distance calculation, obviously. Since currently all calculations are done assuming the points are on the current map (since we only track objectives on the current map).

tl;dr we need to move everything over to Astrolabe calculations and continents/zones before we go about any map-handling.

**Comment by Logonz** [comment]
1.1 is very close, closing to clean up the tracker
